### ISSUE TITLE
Doesn't support releases with non-integer versions

### ISSUE THREAD

**Issue by drnic** [cloudfoundry-incubator/bosh-workspace#37]
I have a release with version v0.1 but this fails for bosh-workspace:

```
$ bosh prepare deployment
Validation errors:
- { releases => At index 1: { version => Object 0.1 doesn't validate against any of #<Membrane::Schemas::Class:0x007f12be53c190>, #<Membrane::Schemas::Value:0x007f12be53c0f0> } }
'/home/ubuntu/workspace/deployments/cf-boshworkspace/deployments/cf-aws-tiny.yml' is not valid
```

**Comment by rkoster** [comment]
@drnic can you test if it works with the latest master? I should be fixed in https://github.com/cloudfoundry-incubator/bosh-workspace/commit/b490ac836f88e6b956195bd0b6accf83912f5695

**Comment by rkoster** [comment]
The parsing works but now there are still some issues with `0.1022.0` which should upload `diego-0.1022.0.yml` but instead uploads `diego-0.935.yml`

**Comment by rkoster** [comment]
Fixed in [0.9.0.rc3](https://rubygems.org/gems/bosh-workspace/versions/0.9.0.rc3)
